File: Programetions/Segundo modulo/aula2/atividade4.py
# Exercício 4: Crie um programa que recebe uma string como entrada e verifica se ela começa com a letra 'A'. Imprima "Começa com A" se for verdadeiro, caso contrário, imprima "Não começa com A".

frase = input('Insira uma frase: ').lower()

if frase[0] == 'a':
    print('A Primeira letra é A.')
else:
    print('Não começa com a letra A.')

File: Programetions/Quarto modulo/aula2/practice/atividade4.js
// Exercício 4: Crie um programa que recebe uma string como entrada e verifica se ela começa com a letra 'A'. Imprima "Começa com A" se for verdadeiro, caso contrário, imprima "Não começa com A".

const stringrecebida = "abobora";
const verificar = stringrecebida[0].toLowerCase() === "a"? "Começa com A": "Não começa com A";
console.log(verificar)